app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -259,13 +259,13 @@ const SendAdaPage = ({
     return (
       <div className="wrapper flex-nowrap shrinkable">
         <FormattedAssetItem {...dropdownAssetItem}>
-          {({formattedHumanReadableLabelType, formattedHumanReadableLabelWithIcon}) => {
-            const isFingerprint =
-              formattedHumanReadableLabelType === FormattedHumanReadableLabelType.FINGERPRINT
+          {({formattedHumanReadableLabelVariants}) => {
+            const {type: labelType, label} = formattedHumanReadableLabelVariants
+            const isLabelFingerprint = labelType === FormattedHumanReadableLabelType.FINGERPRINT
             return (
               <Fragment>
-                {formattedHumanReadableLabelWithIcon}
-                {type === AssetFamily.TOKEN && !isFingerprint && (
+                {label}
```

Comment:
given there was originally `formattedHumanReadableLabelWithIcon`, shouldn't this be the `withIcon` variant? Btw, I'd consider naming the `withIcon` property `labelWithIcon` for the sake of readability when used alone, same for `shortWithIcon`

Reply:
Yes, it should be with an icon, fixed. Also refactored labels.